pkg/defaults/defaults.go

```diff
@@ -78,6 +78,9 @@ const (
 	// DefaultMapRoot is the default path where BPFFS should be mounted
 	DefaultMapRoot = "/sys/fs/bpf"
 
+	// DefaultCgroupRoot is the default path where cilium cgroup2 should be mounted
+	DefaultCgroupRoot = "/mnt/cilium_cgroup2/"
```

Comment:
This doesn't seem to be a standard path

Reply:
We just need to make it standard for all distros  :smile:

Reply:
Could probably go under `/var/run/cilium/cgroupv2/`. Afaik the mount for cgroupsv2 would just be another instance pointing the same singleton instance anyway.

Reply:
right, its fine to have multiple mounts for cgroupv2 and the all point to the same instance. Latest ubuntu for example seems to create a mount for cgroupv2. That said /var/run/cilium/cgroupv2 seems fine to me I'll move it there.